projects/authentication/docs/user-access-token.md

```diff
@@ -281,23 +275,23 @@ Authorization: Bearer eyJz93a...k4laUWw
 
 #### More info
 
-publiq uses [Auth0](https://auth0.com/) as its authentication and authorization server. To learn more about the Authorization Code Flow with PKCE, see the [the Auth0 documentation](https://auth0.com/docs/flows/authorization-code-flow-with-proof-key-for-code-exchange-pkce).
+publiq uses [Keycloak](https://www.keycloak.org/) as its authentication and authorization server. To learn more about the Authorization Code Flow with PKCE, see [RFC 7636: Proof Key for Code Exchange](https://oauth.net/2/pkce/).
```

Comment:
Publiq with a capital letter, and same remark regarding Keycloak.

Reply:
is `publiq` writtern with a capital P? :) (it was already there in small caps, and I see regularly see it small caps even on publiq.be)